### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by ejm714**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ version = "0.1.0"
 authors = [
   {name = "DrivenData", email = "info@drivendata.org"}
 ]
-description = "CyFi is a Python package to estimate cyanobacteria density in small, inland water bodies using Sentinel-2 satellite imagery."
+description = "Estimate cyanobacteria density in small, inland water bodies using Sentinel-2 satellite imagery."
 readme = "README.md"
 license = {text = "MIT License"}
 classifiers = [
```
Comment (pjbull) [inline pyproject.toml:14]: Good to add supported Python versions to classifiers for PyPI.

File: pyproject.toml [hunk 2]
```diff
@@ -8,7 +8,7 @@ version = "0.1.0"
 authors = [
   {name = "DrivenData", email = "info@drivendata.org"}
 ]
-description = "CyFi is a Python package to estimate cyanobacteria density in small, inland water bodies using Sentinel-2 satellite imagery."
+description = "Estimate cyanobacteria density in small, inland water bodies using Sentinel-2 satellite imagery."
 readme = "README.md"
 license = {text = "MIT License"}
```
Comment (pjbull) [inline pyproject.toml:13]: I think we want to point at the license file here with `license = { file = "LICENSE" }`

File: pyproject.toml [hunk 3]
```diff
@@ -42,6 +44,7 @@ dependencies = [
   "rioxarray",
   "tqdm",
   "typer",
+  "xarray[accel,io,parallel]"
 ]
```
Comment (pjbull) [inline pyproject.toml:49]: Add urls, e.g.: https://github.com/drivendataorg/cloudpathlib/blob/master/pyproject.toml#L43-L47

File: pyproject.toml [hunk 4]
```diff
@@ -19,11 +19,13 @@ classifiers = [
 requires-python = ">=3.10"
```
Comment (pjbull) [inline pyproject.toml:19]: Add `keywords`, e.g.: https://github.com/drivendataorg/cloudpathlib/blob/master/pyproject.toml#L12-L18

File: .github/workflows/tests.yml [hunk 5]
```diff
@@ -39,7 +39,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: ["3.10", 3.11]
```
Comment (pjbull) [inline .github/workflows/tests.yml:43]: Does "3.11" need quotes?

File: .github/workflows/tests.yml [hunk 6]
```diff
@@ -65,3 +65,20 @@ jobs:
...
         run: |
           make test
+
+      # - name: Upload coverage to codecov
+      #   uses: codecov/codecov-action@v3
+      #   with:
+      #     token: ${{ secrets.CODECOV_TOKEN }}
+      #     file: ./coverage.xml
+      #     fail_ci_if_error: true
+      #   if: ${{ matrix.os == 'ubuntu-latest' }}
```
Comment (pjbull) [inline .github/workflows/tests.yml:75]: Uncomment or remove

**Review by pjbull — COMMENTED** [review]
A few comments on metadata for PyPI, otherwise lgtm!

File: .github/workflows/tests.yml (hunk 5 above)
Comment (ejm714) [reply .github/workflows/tests.yml:43]: no, 3.10 has quotes just so it doesn't show up as 3.1 in the github actions sidebar

**Pull request merged**